build/configs/rtl8721csm/scripts/rlx8721d_img2_is.ld

```diff
@@ -142,16 +215,26 @@ SECTIONS
 		*(.bss*)
 		*(COMMON)
 		__bss_end__ = .;
-		_ebss = ABSOLUTE(.);
+
+	} > BD_RAM_NS
+
+	.ram_image2.nocache.data :
+	{
+		. = ALIGN (32);
+		__ram_nocache_start__ = .;
+		*(.bdsram.data*)
+		. = ALIGN (32);
+		__ram_nocache_end__ = .;
 	} > BD_RAM_NS
 
 	.ram_heap.data : 
 	{
+		. = ALIGN (32);
 		*(.bfsram.data*)
 		end = .;
 		*(.heap.stdlib*)
 		. = . + 4096;
-	
+		_ebss = ABSOLUTE(.);
```

Comment:
When you move ebss here, then we are allocating some 4096 bytes for heap. Why is this space required? In TizenRT, we are not using this heap region. Separate heap is allocated after the bss and idle thread stack.

Reply:
We verified and removed 4096 bytes memory allocated for heap.stdlib from ld script which is not used on TizenRT. Please refer to #4668.